src/components/organisms/leporello-graphic-real-item/leporello-graphic-real-item.jsx

```diff
@@ -54,85 +56,103 @@ export default ({
+              term={t('Dimensions')}
               definition={dimensions}
             />
           </GroupedDefinitionList>
 
           {/* Besitz, Eigentümer und so */}
           <GroupedDefinitionList>
             <GroupedDefinitionList.Entry
-              term={ t('Owner') }
+              term={t('Owner')}
               definition={owner}
             />
             <GroupedDefinitionList.Entry
-              term={ t('Repository') }
+              term={t('Repository')}
               definition={repository}
             />
             <GroupedDefinitionList.Entry
-              term={ t('Location') }
+              term={t('Location')}
               definition={location}
             />
           </GroupedDefinitionList>
 
           {/* Inschriften, Texte und so */}
-         { (signature || inscription || markings || description || provenance)
+          {(signature || inscription || markings || description)
             && (
               <GroupedDefinitionList>
                 {signature && <GroupedDefinitionList.Entry
-                  term={ t('Signature / Dating') }
+                  term={t('Signature / Dating')}
                   definition={signature}
                 />}
                 {inscription && <GroupedDefinitionList.Entry
-                  term={ t('Inscriptions') }
+                  term={t('Inscriptions')}
                   definition={inscription}
                 />}
                 {markings && <GroupedDefinitionList.Entry
-                  term={ t('Stamps, Seals, Labels') }
+                  term={t('Stamps, Seals, Labels')}
                   definition={markings}
                 />}
+              </GroupedDefinitionList>
+            )
+          }
+
+          {/* Beschreibung, Provenienz, Ausstellungen */}
+          {(provenance || exhibitionHistory || description)
+            && (
+              <GroupedDefinitionList>
                 {description && <GroupedDefinitionList.Entry
-                  term={ t('Short description') }
+                  term={t('Short description')}
                   definition={description}
                 />}
                 {provenance && <GroupedDefinitionList.Entry
-                  term={ t('Provenance') }
+                  term={t('Provenance')}
                   definition={provenance}
                 />}
+                {exhibitionHistory && <GroupedDefinitionList.Entry
+                  term={t('Exhibitions')}
+                  definition={exhibitionHistory}
+                />}
               </GroupedDefinitionList>
             )
           }
 
+          {/* Kennungen */}
+          <GroupedDefinitionList>
+            <GroupedDefinitionList.Entry
+              term="CDA ID"
+              definition={inventoryNumber}
+            />
+            {catalogWorkReferences.length > 0
+              && <GroupedDefinitionList.Entry
+              term={t('VD16-No.')}
+              definition={catalogWorkReferences}
```

Comment:
Ist das hier so gewollt?
`catalogWorkReferences` ist ja ein Array und `GroupedDefinitionList.Entry` erwartet eher etwas von einem basic type (number, string usw.).
Durch den "automatischen" Aufruf von `toString` für das Array, erhält man dann eher sowas hier `[object Object],[object Object]`. Die Array-Items komma-separiert.
Um an die `VD16`-Nummer zu kommen müsste man dann wahrscheinlich auch noch überprüfen, ob diese überhaupt in dem `catalogWorkReferences` des realen Objekts überhaupt enthalten ist, oder?

Betreffend der Katalognummern hatte ich in der LeporelloGraphicDetailsItem-Komponente bereits schon mal was umgesetzt gehabt ([Sortiergewichtung](https://github.com/lucascranach/cranach-grafiken-gatsby/blob/3e81503291d4c7f174cee23b7ebbd6e991d1ad73/src/components/organisms/leporello-graphic-details-item/leporello-graphic-details-item.jsx#L45) u. [Mapping für DefinitionList.Entry](https://github.com/lucascranach/cranach-grafiken-gatsby/blob/3e81503291d4c7f174cee23b7ebbd6e991d1ad73/src/components/organisms/leporello-graphic-details-item/leporello-graphic-details-item.jsx#L69)) . Evtl. müsste dort noch `VD16` aufgenommen werden.

Da kann man dann auch sehen, wie der Translation-Funktion noch Key-Value-pairs für String-Interpolationen mitgegeben werden können (`{{catalogWorkReferenceName}}-No` -> `t('{{catalogWorkReferenceName}}-No', { catalogWorkReferenceName: 'wert' })`).

Reply:
ja, das ist bullshit so … das müssten wir uns noch mal zusammen anschauen bitte

Reply:
Können wir machen.

Du hast aber ja schon mal einen neuen Change vorgenommen, bei dem nun alle Items des `catalogWorkReferences`-Arrays zu einem `GroupedDefinitionList.Entry` gemappt werden.

Sieht für mich erstmal nach einem guten Zwischen-Step aus.
Hätte jetzt gesagt, dass man diese Konversation (und die folgende) erstmal "resolven" kann. Würde das für dich so passen?